readme.md

```diff
@@ -25,7 +25,36 @@ import "github.com/waifuvault/waifuVault-go-api/pkg"
 
 // then init the API
 
-api := waifuVault.Api{}
+api := waifuVault.NewWaifuvaltApi(http.Client{})
+```
+
+### Contexts
+
+Each function on the Waifuvalt API takes a context. This can be `nil`
```

Comment:
Nah it cannot.

If user doesnt really cares about context, it's better for them pass `context.TODO()` or `context.Background()`, so it shouldn't be care of library.